### DISCUSSION TITLE
Authentication Failed

### DISCUSSION THREAD

**Discussion by NotWhistle** [R2Northstar/Northstar#226] [Q&A]
When I try to load the Northstar client every time I press Northstar to load the multiplayer it said authentication failed and doesn't give me a way to fix it. Does anyone know a solution?

**Comment by GeckoEidechse** [comment]
As said in [the pinned isse](https://github.com/R2Northstar/Northstar/issues/124) as well as the announcement channel in the [Northstar Discord server](https://northstar.tf/discord), Northstar is currently down while we fix some security exploits (coming from the base game).

We will announce once the servers are back up.
